src/components/checkbox/_type/checkbox_type_button.bemhtml

```diff
@@ -0,0 +1,27 @@
+block('checkbox').mod('type', 'button')(
+    content()(function() {
+        const React = require('react');
+        const Button = require('../src/components/button/button');
+
+        const ctx = this.ctx,
+            mods = this.mods;
+
+        const originalContent = applyNext();
```

Comment:
It's right if previous level is bem-components, but if you add another level before this template we can get truble